### DISCUSSION TITLE
Parametri qualità aria

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#2342] [GENERALE - Discussioni generiche]
Ciao a tutti, 
ho recentemente installato un sensore qualità aria  tuya TS0601_smart_air_house_keeper.
dove posso reperire valori di riferimento dei cari parametri (Co2, VOC, etc etc)  per creare dashboard e allert.
avete link di riferiemento?

**Comment by indomus** [comment]
> Ciao a tutti, ho recentemente installato un sensore qualità aria tuya TS0601_smart_air_house_keeper. dove posso reperire valori di riferimento dei cari parametri (Co2, VOC, etc etc) per creare dashboard e allert. avete link di riferiemento?

hm no, prova su wikipedia

**Reply by evirci** [reply]
è arrivato oggi pure a me, ma ho cercato i valori online ;)
Tipo la CO2 deve essere inferiore a 1400 ppm (parts per million) è buona. Tra 1400-2000 ppm è media. Superiore a 2000 ppm è pessima. Normalmente è sui 400ppm

**Reply by Matty1184** [reply]
per il co2 ho trovato tutte le guide, nessun problema.
per quanto riguarda pm2.5, VOC, Formaldeide invece non ho trovato parametri di riferimento attendibili.

**Reply by indomus** [reply]
> per il co2 ho trovato tutte le guide, nessun problema. per quanto riguarda pm2.5, VOC, Formaldeide invece non ho trovato parametri di riferimento attendibili.

guarda la tabella a metà articolo: https://indomus.it/componenti/amazon-smart-air-quality-monitor-multi-sensore-di-qualita-dellaria/
